packages/vscode-extension/package.json

```diff
@@ -0,0 +1,194 @@
+{
+  "private": true,
+  "version": "0.0.1",
+  "name": "moon",
+  "publisher": "moonrepo",
+  "displayName": "moon",
+  "description": "Control panel and console for moon.",
+  "markdown": "github",
+  "keywords": [
+    "moon",
+    "moonrepo",
+    "build",
+    "system",
+    "ci",
+    "cd",
+    "console"
+  ],
+  "engines": {
+    "vscode": "^1.72.0"
```

Comment:
The engine support seems to be a bit high, as this is the "latest" at this time and can be lowered to `^1.66.0` which supports all the APIs used